scaf

```diff
@@ -61,34 +75,99 @@ party_popper() {
     echo
 }
 
+start_challenge_oauthflow() {
+    # Kick off scaf challenge oauthflow process
+    CHALLENGE_SCRIPT="./scaf-challenge-oauth.sh"
+    if [ -f "$CHALLENGE_SCRIPT" ]; then
+        rm $CHALLENGE_SCRIPT
+    fi
+    # Download challenge script and start oauth flow
+    echo "Downloading scaf challenge oauthflow script from $SCAF_CHALLENGE_SCRIPT_URL..."
+    curl -L $SCAF_CHALLENGE_SCRIPT_URL -o $CHALLENGE_SCRIPT
```

Comment:
Can we do this without the download of the script? Is there a downside to embedding the `challenge.sh` script in the `scaf` one?